games-engines/openscenegraph/openscenegraph-3.6.5.recipe

```diff
@@ -67,18 +62,19 @@ REQUIRES="
 	lib:libgif$secondaryArchSuffix
 	lib:libGL$secondaryArchSuffix
 	lib:libglib_2.0$secondaryArchSuffix
-#	lib:libgstreamer_1.0$secondaryArchSuffix
-	lib:libhalf_2_4$secondaryArchSuffix
-	lib:libIlmImf_2_4$secondaryArchSuffix
+	lib:libiex_3_2$secondaryArchSuffix
+	lib:libilmthread_2_4$secondaryArchSuffix
+	lib:libimath_2_4$secondaryArchSuffix
```

Comment:
I don't get this, 2 different openEXR versions?

If this is only to fix the cmake build, isn't enough to add one line in BUILD() for this?

Reply:
Giving up on this ... you remove (even if disabled) parts that were put in for a reason, switch stuff which is not needed ...

Reply:
This recipe has everything building properly and prepped for OpenEXR 3.2. I pushed back to OpenEXR 2.4 before I merged the OpenEXR change. We can cancel it.